packages/shared/src/components/comments/CommentContainer.tsx

```diff
@@ -145,8 +150,23 @@ export default function CommentContainer({
                 @{comment.author.username}
               </TruncateText>
             </ProfileLink>
-            <div className="mx-2 h-0.5 w-0.5 bg-text-quaternary" />
+            <Separator />
             <CommentPublishDate comment={comment} />
+            {topReader && (
+              <>
+                <Separator />
+                <SimpleTooltip
+                  content={formatDate({
+                    value: topReader.issuedAt,
+                    type: TimeFormatType.TopReaderBadge,
+                  })}
+                >
+                  <TruncateText>
+                    Top reader in {topReader?.keyword?.flags?.title}
```

Comment:
Not sure, but should the tag link to tag page maybe?

Reply:
I can ask product

Reply:
Leaving it out.
https://dailydotdev.slack.com/archives/C078JGTQ9AT/p1730720015760399